components/function-controller/internal/controllers/kubernetes/handler.go

```diff
@@ -1,4 +1,4 @@
-package controllers
+package kubernetes
```

Comment:
We should after 1.12 talk about creating a separate controller for all similar tasks, i.e. configuration propagation ConfigMaps, Secrets, ServiceAccounts, Roles, RoleBindings to Namespaces :)